src/py/flwr/server/strategy/fedadagrad.py

```diff
@@ -37,15 +37,15 @@
 from .fedopt import FedOpt
 
 
-# pylint: disable=too-many-locals
+# flake8: noqa: E501
```

Comment:
This should be enough, right? That's how we have it in other parts of the codebase
```suggestion
# noqa: E501
```

Reply:
@danieljanes only if it's for a specific line. As it's in the doc string, I can't really put it into that line.